vendor/vendor.json

```diff
@@ -106,6 +106,12 @@
 			"version": "v12.5.0-beta",
 			"versionExact": "v12.5.0-beta"
 		},
+		{
+			"checksumSHA1": "l4BaDOG5showWuLiGBjsY4eVhtk=",
+			"path": "github.com/Azure/azure-sdk-for-go/services/iothub/mgmt/2017-07-01/devices",
+			"revision": "1e334c402ea1460704b0263e5d188f28ad946ce1",
+			"revisionTime": "2018-02-16T17:41:56Z"
```

Comment:
can we vendor this using the `v12.5.0-beta` tag? I believe that should mean we can downgrade to `v10` of AutoRest too - you can do this via:

```
govendor update github.com/Azure/azure-sdk-for-go/services/iothub/mgmt/2017-07-01/devices@=v12.5.0-beta
```